ChromecastSampleApp/ChromecastSampleApp/OOCastManagerFetcher.m

```diff
@@ -11,6 +11,6 @@
 
 @implementation OOCastManagerFetcher
 +(OOCastManager*) fetchCastManager {
-  return [OOCastManager getCastManagerWithAppID:@"D4C7BD4A" namespace:@"urn:x-cast:ooyala"];
+  return [OOCastManager getCastManagerWithAppID:@"B3560E04" namespace:@"urn:x-cast:ooyala"];
```

Comment:
I just think that as it was custom receiver, then let's keep custom receiver.  Custom receiver, unlike default one, allows controlling v3 version.  Besides, custom receiver allows user to modify receiver code, whereas default receiver allows only to customize icons.  We actually may discontinue support for default receiver altogether at some point.